tendrl/ceph_integration/objects/definition/ceph.yaml

```diff
@@ -210,9 +222,21 @@ namespace.ceph:
           enabled: true
           inputs:
             mandatory:
-              - Rbd.pool_id
               - Rbd.name
               - Rbd.size
+            optional:
```

Comment:
CC: @anivargi @cloudbehl @gnehapk How does the API and the UI handle optional params